service/k8sclient/internal/kubectl/kubectl.go

```diff
@@ -283,3 +322,55 @@ func run(ctx context.Context, kubectlCmd []string, args []string, stdin interfac
 	l.Debug(errBuf.String())
 	return outBuf.Bytes(), err
 }
+
+// RunDaemon runs kubectl command with args and assigns the process to daemonCmd
+// so we can later kill it. This call blocks. StopDaemon has to be run when daemon
+// is no longer needed. Concurrent calls to this method blocks until the current
+// daemon is stopped.
+func (k *KubeCtl) RunDaemon(ctx context.Context, args ...string) error {
```

Comment:
it looks supercomplicated. But I have no other idea that we can for minikube.

Reply:
I added more complexity to make it possible to use this `kubectl proxy` in concurrent requests - basically using different port for proxy for each request.

Reply:
I changed the name to RunProxy, because it's what we do. I could then implement more details right in the method. PTL